icu4c/source/data/cldr-icu-readme.txt

```diff
@@ -161,11 +161,14 @@ ant all
 #ant jar
 
 # 3. Configure ICU4C, build and test without new data first, to verify that
-# there are no pre-existing errors (configure shown here for MacOSX, adjust
-# for your platform).
+# there are no pre-existing errors.
+# If you do not want to re-run configure and make in step 12 below, then define
+# ICU_DATA_BUILDTOOL_OPTS=--include_uni_core_data here in order to generate the
```

Comment:
Technically I think it's better to build and test ICU4C without including the unicore data.

Reply:
Sure. But to expedite things maybe you can make that update as part of the CLDR integration process which I am now turning over to you.

Reply:
OK